### ISSUE TITLE
Modelica package structure for whole DHC system model

### ISSUE THREAD

**Issue by AntoineGautier** [urbanopt/geojson-modelica-translator#122]
This issue is for iterating and agreeing on the package structure to adopt for a whole DHC system model generated by GMT.
It would be good if the translator used the same hierarchy as the library (except that `Applications.DHC` can be dropped as it generates a Modelica package that lives outside the library). 
Also, if a component can directly be instantiated, it should do so, no need to extend it first as this would just add complexity. 
A use case is however that someone wants to reuse say a substation model but has to modify its structure. For this, it would be useful to optionally **copy** the model so the user can edit it in his/her package as the library should not be edited by a user.

**Comment by nllong** [comment]
Yes, this is a great ticket to start the discussion. The directory structure is scaffolded based on early discussion that we had with you all. We can change it as needed.

In general, we use models directly from the MBL without copying them. There is a couple of cases where the models are not in the MBL (yet?) such as the CoolingIndirect.mo component. These files are copied into the scaffolded project as we wait for MBL to "catch up". 

The last case is when we have to templatize a model that exists in the MBL to handle extensibility that is difficult in Modelica. These files have a .mot and .most extensions. 

We can discuss the project structure more on a technical call.

**Comment by kflemin** [comment]
closing as stale. Reopen if this is still an issue.
